README.md

```diff
@@ -14,17 +13,69 @@
+
+![File listing](.images/xcode02.png)
+
+## Storyboards
+
+Storyboards (`.storyboard` files) are where you lay out the graphical interface of your iOS program. By default, an Xcode project includes both a `LaunchScreen.storyboard` file and a `Main.storyboard` file. `LaunchScreen.storyboard` is where you lay out the interface that you see when your program is in the process of launching—its "splash screen". These are important, but you won't focus on `LaunchScreen.storyboard` in this lesson.
+
+`Main.storyboard` is the main graphical interface of your program. It is the first thing your user will see after your program is done launching. You can think of it as the "entry point" of your iOS program. It is in this storyboard that you will place the initial controls—text boxes, buttons, toolbar items, and so forth—that will allow your user to navigate to the rest of your program. Some simple programs may _only_ have this single interface.
+
+Open up `Main.storyboard` by clicking once on it in the file listing. You'll see it open in Interface Builder. Right now, it's pretty plain—just a large white box!
+
+![`Main.storyboard`](.images/xcode03.png)
+
+Right now, there is one _object_ on the storyboard: A blank white box called **View Controller**. A view controller is an object that is responsible for drawing the graphical interface, as well as managing interactions with it (touches, swipes, pokes, and so forth). You will be learning _a lot_ more about view controllers in the upcoming lessons, but for now, just know that it is responsible for managing how the interface _looks_.
+
+Notice that on the left side of the view controller's rectangle is a grey arrow pointing to it (you may have to scroll a bit to the left to see it). That arrow indicates that this view controller is the entry point of your application. It's the first thing a user will see when they run your application.
+
+You should also notice that on the left hand side of the Interface Builder window, next to the file listing, is a list of items. This area is the _object listing_. The topmost item is labeled **View Controller Scene**. Click once on this item. You should notice that the big blank view controller rectangle in the editing window is highlighting. **View Controller Scene** in the object listing corresponds to that view controller rectangle in the editing portion.
+
+![View Controller](.images/xcode04.png)
+
+### Scenes
+
+Have you ever seen a storyboard for a movie? It consists of drawings for key scenes of a movie, and shows how the story flows and what the movie will look like at each scene. iOS development uses a similar model to lay out graphical interfaces. An iOS app consists of one or more _screens_ which display text, images, icons—anything you want to show to the user. You move between these screens using controls like buttons.
+
```

Comment:
One thing to add here could be referencing something like the Contacts app people are familiar with. You have your list of contacts on one screen and by tapping on any individual within that list, we are presented with another screen.